app/components/avatar_component.rb

```diff
@@ -0,0 +1,25 @@
+class AvatarComponent < ViewComponent::Base
```

Comment:
This prevents me from setting a specific size for the gravatar. Also, if we are using the gravatar, do we want to pass in the size (converted to px) in the `user_gravatar_url` call?

Reply:
We are already using this today, I just took it out of the module it was in. If you’d like to look into that implementation, we should do it in another PR